resultsdbupdater/utils.py

```diff
@@ -446,6 +446,14 @@ def handle_ci_umb(msg):
         # Old topics are allowed for now.
         msg.log.warning(e)
 
+    if outcome == 'ERROR':
+        error_reason = msg.get('error', 'reason')
```

Comment:
`'reason'` is a bad default value. An empty string would be better. But it's considered a required field in the schema: https://pagure.io/fedora-ci/messages/blob/master/f/schemas/error.yaml

Should messages be rejected instead?

Reply:
This is not default value, 'reason' is subfield of 'error' in this case. If the fields are not available, the message will be rejected.

It's confusing, I should rename `Message.get`, or split it to `Message.require` and Message.optional`.

Reply:
Oh! It's a path?

Reply:
Yup, I really need to change the name.